src/ui/map.js

```diff
@@ -295,6 +295,8 @@ const defaultOptions = {
  *  * [Albers](https://en.wikipedia.org/wiki/Albers_projection) equal-area conic projection as `albers`
  *  * [Equal Earth](https://en.wikipedia.org/wiki/Equal_Earth_projection) equal-area pseudocylindrical projection as `equalEarth`
  *  * [Equirectangular](https://en.wikipedia.org/wiki/Equirectangular_projection) (Plate Carrée/WGS84) as `equirectangular`
+ *  * Globe, a 3D spherical projection, as `globe`
```

Comment:
As above, we should reconsider projection order

Reply:
This is alphabetical, Maybe we list Globe and Mercator first and the rest in alphabetical? I think we should standardize this with all projection lists